catalog/ui/catalog-ui-search/src/main/webapp/component/navigation-right/navigation-right.hbs

```diff
@@ -11,23 +11,23 @@
  *
  **/
  --}}
+
 <button class="navigation-item item-help is-button" title="Shows helpful hints in the current context.">
     <span class="fa fa-question"></span>
 </button>
 <button class="navigation-item item-settings is-button" title="Shows settings for the application.">
     <span class="fa fa-cog"></span>
 </button>
-<button class="navigation-item item-alerts is-button" title="Shows notifications.">
+<button class="navigation-item item-alerts is-button" style="margin-right: 15px;" title="Shows notifications.">
```

Comment:
i think we want to avoid inline styling if we can; might be best to extract this into the `.less` file for the `navigation-right` component